tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/logout_bzlite.py

```diff
@@ -0,0 +1,13 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.bzlite.app import BugzillaLite
+
+class TestLogout(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+    def test_logout_BugzillaLite(self):
+        bugzilla_lite = BugzillaLite(self.marionette)
+        bugzilla_lite.launch()
+        bugzilla_lite.logout()
```

Comment:
In our continous integration, we add `--restart` at the end of the command line. `--restart` clears the phone. Hence, we need to login in this test before login out .